.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: release
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permission:
+      contents: read|write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Release
+        id: release
+        run: |
+          npm run release
+          echo ::set-output name=VERSION::$(jq -r .version ./package.json)
+      - name: Create pull request
```

Comment:
does it automatically creates a git tag? I see that a branch is created but I am not sure about the tag.

Reply:
It does not create the git tag. This will happen in another PR I am working on.

Reply:
let's rename the PR to `chore: automate release PR creation`